projects/aca-content/src/lib/components/sidenav/directives/expansion-panel.directive.ts

```diff
@@ -58,10 +58,10 @@ export class ExpansionPanelDirective implements OnInit, OnDestroy {
   constructor(private store: Store<any>, private router: Router, private expansionPanel: MatExpansionPanel) {}
 
   hasActiveLinks() {
-    if (this.acaExpansionPanel && this.acaExpansionPanel.children) {
+    if (this.acaExpansionPanel && !this.acaExpansionPanel.children) {
```

Comment:
this does not make any sense, why you are changing this function? 
as per your change, it checks if there no children, and then enumerate the missing children? this is plain stupid